src/features/script-load.js

```diff
@@ -63,6 +63,15 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
     return autoImportRegistration;
   }
   var loader = this;
+
+  var inlineScriptContent = inlineScripts[url]
+  if (typeof inlineScriptContent === "string") {
+    if (inlineScriptContent.indexOf("//# sourceURL=") < 0)
+      inlineScriptContent += "\n//# sourceURL=" + url
+    ;(0, eval)(inlineScriptContent)
```

Comment:
If I understand things correctly, your code here requires the user to call `System.import('https://example.com/_inline_script_0')` in order for the `<script type="systemjs-module"></script>` to be executed? If so, I don't think it's a good solution for the following reasons:

1. The `_inline_script_0` part is a magic string that would not be obvious for people learning systemjs
2. Correlating an inline module to a URL is not something that browsers do and, as far as I know, there's no precedent for it.
3. The user would probably expect the inline module to be executed automatically, and be confused when it isn't and requires them to call `System.import('https://example.com/_inline_script_0')` manually to execute it.

Reply:
> your code here requires the user to call System.import('https://example.com/_inline_script_0')

The code is auto executed when systemjs collects the script tags and calls System.import 

https://github.com/systemjs/systemjs/blob/4da6ac77f78437f1c193f981efe90279016b9724/src/features/import-maps.js#L45

Reply:
Ah I missed that, makes sense.